jenkinsfiles/canary

```diff
@@ -41,6 +41,61 @@ pipeline {
                 }
             }
         }
+
+        stage('Update IM Play dev instance') {
```

Comment:
This could be "canary" instances instead of "play" instances of we're being pedantic 🙂

Reply:
I'll just remove the dev since the context is already given from the pipeline. I would it concise, not pedantic :slightly_smiling_face:  Thanks :pray: